coalib/output/printers/LogPrinterMixin.py

```diff
@@ -0,0 +1,45 @@
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
```

Comment:
> Oh, its being used by other projects actually. One of the bears is using it in coala/coala-bears.

Yep we should get rid of that.

As for providing flexibility: we have to be careful, flexibility means complication and this whole thing is to remove complication, if we can remove the whole log_printer stuff that'd be simpler for people to understand because they already know builtin logging. There must be a simple way to get builtin logs for testing, right?

```
lasse@lsmob ~/prog/coala/coala (git)-[log] % git grep log_printer
coalib/processes/DebugProcessing.py:    ``log_printer``).
coalib/processes/DebugProcessing.py:        Gets local ``log_printer`` from function that created this instance.
coalib/processes/DebugProcessing.py:        there is a ``self.log_printer``. Then `item` is just sent to logger
coalib/processes/LogPrinterThread.py:    def __init__(self, message_queue, log_printer=None):
coalib/processes/LogPrinterThread.py:        self.log_printer = log_printer
coalib/processes/LogPrinterThread.py:                if self.log_printer is not None and getattr(
coalib/processes/LogPrinterThread.py:                        self.log_printer, 'log_message'):
coalib/processes/LogPrinterThread.py:                    self.log_printer.log_message(elem)
tests/processes/LogPrinterThreadTest.py:        log_printer = TestPrinter()
tests/processes/LogPrinterThreadTest.py:        self.uut = LogPrinterThread(log_queue, log_printer)
```